test/test_todo.py

```diff
@@ -0,0 +1,58 @@
+from flask import url_for
+
+from .conftest import create_package, create_group, create_issue, create_advisory, DEFAULT_GROUP_ID, DEFAULT_ADVISORY_ID, DEFAULT_ISSUE_ID
+from app.model.enum import Publication, Severity, Remote
+from app.model.cve import issue_types
+from app.view.todo import get_todo_data
+
+
+def todo_data(todo_data, expected):
```

Comment:
can we name this something like assert_todo_date so we know what the function actually does?

Reply:
That's the plan, this PR is mostly work in progress still.